package.json

```diff
@@ -49,14 +49,15 @@
   },
   "dependencies": {
     "broccoli-funnel": "^3.0.2",
+    "ember-auto-import": "^2.6.3",
     "ember-cli-babel": "^7.26.11",
     "ember-cli-typescript": "^5.1.1"
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.21.3",
     "@babel/plugin-proposal-decorators": "^7.21.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.0.1",
+    "@ember/string": "^4.0.0",
```

Comment:
This fixes an issue introduced in #547. The project had a direct dependency on `^3.0.1` of `@ember/string` but a resolution forcing all transitive dependencies to resolve to `^4.0.0`. Please let me know if you prefer this to be pulled out to a separate PR.